app/src/main/java/com/hammer/talkbbokki/presentation/topics/TopicListScreen.kt

```diff
@@ -160,29 +188,31 @@ fun SelectBtn(viewCnt: Int, onCardClicked: () -> Unit, modifier: Modifier) {
         shape = RoundedCornerShape(8.dp)
     ) {
         Text(
-            text = if (viewCnt >= 3) "광고 보고 뽑기" else "이 카드 뽑기",
-            style = TalkbbokkiTypography.button_large,
-            color = Color.White
+            text = if (isOpened)
+                stringResource(R.string.list_re_pick_btn)
+            else if (todayViewCnt >= 3)
+                stringResource(R.string.list_ad_pick_btn)
+            else
+                stringResource(R.string.list_pick_btn),
+            style = TalkbbokkiTypography.button_large, color = Color.White
```

Comment:
🚫 **[ktlint]** <argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)